templates/hotgridPopupToolbar.jsx

```diff
@@ -0,0 +1,80 @@
+import Adapt from 'core/js/adapt';
+import React from 'react';
+import { classes } from 'core/js/reactHelpers';
+
+export default function HotgridPopupToolbar(props) {
+  const a11yConfig = Adapt.config.get('_accessibility');
+  const ariaLabels = Adapt.course.get('_globals')._accessibility._ariaLabels;
+
+  const {
+    onCloseClick,
+    onControlClick,
+    shouldEnableBack,
+    shouldEnableNext,
+    itemCount,
+    _hidePagination
+  } = props;
+
+  return (
+    <div className={classes([
```

Comment:
Strictly speaking the DOM order should reflect the visual order for accessibility reasons. So should render at the top rather than the bottom of the pop up items

Reply:
Done.